fvtest/jitbuildertest/Makefile

```diff
@@ -36,7 +36,8 @@ OBJECTS := \
 	WorklistTest \
 	IfThenElseTest \
 	CallReturnTest \
-	FieldNameTest
+	FieldNameTest \
+  ConvertBitsTest
```

Comment:
Is there a spacing issue here or is it just GitHub?

Reply:
Nope, definitely a spacing issue :sweat_smile:. Thanks for pointing it out. Fixed!